docs/mms/index.md

```diff
@@ -0,0 +1,91 @@
+# Model Server for Apache MXNet (incubating)
+
+Model Server for Apache MXNet (incubating), otherwise known as MXNet Model Server (MMS), is an open source project aimed at providing a simple yet scalable solution for model inference. It is a set of command line tools for packaging model archives and serving them. The tools are written in Python, and have been extended to support containers for easy deployment and scaling. MMS also supports basic logging and advanced logging with AWS CloudWatch integration.
+
+
+## Multi-Framework Model Support with ONNX
+
+While the name implies that MMS is just for MXNet, it is in fact much more flexible, as it can support models in the [ONNX](https://onnx.ai) format. This means that models created and trained in PyTorch, Caffe2, or other ONNX-supporting frameworks can be served with MMS.
+
+To find out more about MXNet's support for ONNX models and using ONNX with MMS, refer to the following resources:
```

Comment:
There are some tutorials on exporting PyTorch models to ONNX, should we link it from the work PyTorch models?